### ISSUE TITLE
Plan for consolidation of our repos

### ISSUE THREAD

**Issue by cricketsloan** [DataBiosphere/administration#64]
We have many versions of things in BD2K and DataBiosphere.  We should be consolidating to DataBiosphere.

**Comment by cricketsloan** [comment]
We have 5 groups to consider

1) Can move to DataBiosphere 

2) Is part of the Boardwalk discussion

3) Redwood platform related, we would like to move en masse to CGP

4) There are candidates for closure (archive or delete)

5) There are tons of other things that are out of our scope

**Comment by david4096** [comment]
https://github.com/DataBiosphere/dss-azul-indexer is a fork of bd2k, I think if you delete the bd2k repo, one of the existing forks becomes the new root.

https://help.github.com/articles/what-happens-to-forks-when-a-repository-is-deleted-or-changes-visibility/

**Comment by mikebaumann** [comment]
For the Commons POC work to date, we have forked the relevant components (including `dss-azul-indexer`) from `BD2KGenomics` to `DataBiosphere`, and put the Commons work on a `feature/commons` branch off of develop. The plan is to then then push this branch from `DataBiosphere` back to `BD2KGenomics`, then move the repo (including the `feature/commons` branch) to `DataBiosphere`. This was done, in part, to avoid impacting the HCA work being done in this repos on `BD2KGenomics`, as they had a big demo about the same time we did.
